packages/pdf-api/src/draft/draft.controller.ts

```diff
@@ -84,4 +84,21 @@ export class DraftController {
 
     return draftDetails;
   }
+
+  @Get('users/:userName/drafts')
+  public async getUserDrafts(
+    @Param('userName') userName: string,
+  ): Promise<DraftDetails[]> {
+    logger.debug('getUserDrafts', { userName });
+
+    const drafts = await this.draftService.getUserDrafts({
+      userName,
+    });
+
+    if (!drafts) {
+      throw new NotFoundException('drafts not found');
```

Comment:
not having any drafts should not result in an error, usually we just return empty list in this case